scripts/build/Platform/Windows/env_windows.cmd

```diff
@@ -18,6 +18,17 @@ IF NOT "%COMMAND_CWD%"=="" (
     CD %COMMAND_CWD%
 )
 
+REM Jenkins reports MSB8029 when TMP/TEMP is not defined, define a dummy folder
+IF NOT "%TMP%"=="" (
+    IF NOT "%WORKSPACE_TMP%"=="" (
+        SET TMP=%WORKSPACE_TMP%
+        SET TEMP=%WORKSPACE_TMP%
+    ) ELSE (
+        SET TMP=%cd%/temp
+        SET TEMP=%cd%/temp
+    )
+)
+
```

Comment:
Do we still have this issue on Jenkins with TEMP not being defined?

Reply:
yeah, but it was solved differently for the windows installer, so I am trying to consolidate it